src/database/keyvalue.rs

```diff
@@ -367,6 +367,10 @@ impl Database for Tree {
             Ok(val)
         })
     }
+
+    fn flush(&mut self) -> Result<usize, Error> {
+        Ok(Tree::flush(self)?)
```

Comment:
Would we use the "bytes written" value in some way? This will also force library users implementing their own DB's flush to return some number.

If it's not super useful maybe we shouldn't expose this as general API just because sled does it?